addons/addon-base/packages/serverless-go-build-tools/README.md

```diff
@@ -7,7 +7,7 @@ This package is a Serverless plugin that will build golang code and upload it to
 #### Tools
 
 - Node 14.x or later
-- Go 1.13.7 or later
+- Go 1.18 or later
```

Comment:
We probably should keep the version fixed here. "or later" suggests any later version would work, even major version updates.